prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,72 @@
+"""
+Taxi simulator
+Estimate: 30 mins
+Actual: 70 mins
+"""
+from prac_09.silver_service_taxi import SilverServiceTaxi
+from prac_09.taxi import Taxi
+
+MENU = 'q)uit, c)hoose taxi, d)rive\n>>> '
+
+
+def main():
```

Comment:
Just missing a docstring here

Reply:
I appear to have a docstring there but I'll add the menu choices to make it more clear. Thanks!

Reply:
I appear to have a docstring but I will add the menu to make this more clear. Thanks!